### PR TITLE
#169817880  get a specific trip request

### PR DESCRIPTION
**Pull request by shemaeric1234**
### What does this PR do?

This PR will help a trip requestor and his or her manager
- To be able to post a comment to a travel request
- To be  able to see the posted comment within the thread

### Description of Task to be completed?

As a user or manager, I should be able to get a specific trip request related to me

### How should this be manually tested?

After clone repository goes to the root of the project open path into command run the npm install and once setup is done click npm test.

Using Postman to test endpoints:
after sign in you will get a token
Then
Key: Content-Type value: application/JSON
To test authentication, add this to the header: Bearer <TOKEN>
 *Add a comment*
using POST request
visit: URL localhost:3000/api/v1/tripRequests/{tripRequestID}
after, you will get a response

#### Any background context you want to provide?

Only for someone who has an account

### What are the relevant pivotal tracker stories?
[#169817880](https://www.pivotaltracker.com/story/show/169817880)

### Screenshots (if appropriate)

### CODE REVIEW

**Review by VedaDom — APPROVED** [review]

**Comment by VedaDom** [comment]
I have approved this PR, but you need to solve these conflicts as soon as possible.

**Review by jo-rdan — APPROVED** [review]
You did a good job eric. kudos!

File: src/routes/trip.route.js [hunk 1]
```diff
@@ -178,4 +179,33 @@ router.patch(
   requestMiddleware.checkIfAlreadyChanged,
   tripController.updateTripRequestStatus
 );
+
+/**
+ * @swagger
+ *
+ * /trip/trip-requests/{tripRequestID}:
```
Comment (mujohn26) [inline src/routes/trip.route.js]: Good work Eric but I don't think the route will be like this you have to remove that `/trip` if you are dealing with trip requests.

Comment (mujohn26) [inline src/routes/trip.route.js]: Good work Eric but I don't think the route will be like this you have to remove that `/trip` if you are dealing with trip requests.

**Review by mujohn26 — APPROVED** [review]

**Review by bdushimi — APPROVED** [review]

**Pull request merged**